lib/paper_trail/events/base.rb

```diff
@@ -218,7 +218,7 @@ def notable_changes
       end
 
       # @api private
-      def notably_changed
+      def notably_changed # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
Rubocop reported this method but it appears too complex to refactor for this PR.  What do you think of disabling for this method?

Reply:
I assume this is because of recent changes in rubocop, see here: https://github.com/rubocop-hq/rubocop/pull/8149.

They also bumped the default from 6 to 7.

`paper_trail` does set a lower goal. Do you by any chance remember how high complexity was? But I agree this is too much for this PR. But rather than disabling it, I'd increase it in `.rubocop_todo.yml`.

https://github.com/paper-trail-gem/paper_trail/blob/fa8b1b270de434b882571b85ad00483817b923cb/.rubocop_todo.yml#L12-L13

I'm a bit hesitant to change it, since @jaredbeck has set these rules and goals. WDYT, @jaredbeck?